src/aligners/tests.rs

```diff
@@ -383,4 +383,183 @@ mod dt {
             }],
         ));
     }
+    //HomoPolymer tests
```

Comment:
Can you fix the comments below on these tests, and them copy them over to the other two test modules. In particular, paste it just below the other 2 occurrences of `del_asymmetric_affine_cost`. I will try to figure out how to clean this up and prevent the duplication